src/reaper_osara.cpp

```diff
@@ -783,16 +783,17 @@ int getStretchAtPos(MediaItem_Take* take, double pos, double itemStart,
 	// Stretch marker positions are relative to the start of the item and the
 	// take's play rate.
 	double posRel = (pos - itemStart) * playRate;
-	if (posRel < 0)
+	double posRelAdj = posRel-0.00003; // approximately 1 sample at 48000. 
```

Comment:
Given that we use this magic value twice, I think it should be a constant. A constant just inside this function is fine. Something like:
`const double STRETCH_FUZZ_FACTOR = 0.00003; // approximately 1 sample at 48000. `

Reply:
Ok, I'll make it a constant.  It could probably be a much smaller number, I'll experiment with that.